mautic/mautic
# Fix filemanager absolute url

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

Noticed interested bug in file manager. This PR fixed it.


#### Steps to test this PR:

See video how to reprouce it: http://youtu.be/POHK5LQ5PBg?hd=1

1. Load up [this PR](https://mautibox.com).
2. Go to emails, try create email.
3. Edit text slot and try insert image - upload new image by file manager

![image](https://user-images.githubusercontent.com/462477/95117920-a1978180-0749-11eb-8d67-6d42e767dbbe.png)

4. Click on new uploaded image 
5. After insert to editor, check source of it
6. Without patch URL of image look like `/staging/efconnect?cmd=file&amp;target=fls1_bXRjLlBORw`. We expected absolute URL in format `https://mautibox.com/staging/media/images/mautic_logo_db64.png`
